apps/workflows/tests/test_endpoints.py

```diff
@@ -374,6 +376,8 @@ def mock_create_comment(self, issue_key: str, body: str):
             **headers,
         )
         assert response.status_code == 400
+        # must reset ACLs to access Flaw
+        set_user_acls(settings.ALL_GROUPS)
```

Comment:
On every test, the default is to bypass ACLs by using a global auto-use fixture, after using the built-in client however the ACLs are reset to no ACLs, so the following database lookup `flaw = Flaw.objects.get(pk=flaw.pk)` will fail because no ACLs are set.

By setting `set_user_acls` here we can access the Flaw object below.

Reply:
I see! Thank you.